docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,66 @@
+==============
+Atop connector
+==============
+
+Overview
+--------
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System & Process Monitor) Linux server performance analysis tool.
+
+In order to use this connector, the machine on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
```

Comment:
and available on the path I assume.

Reply:
Yes.
The location of the tool must be specified through the setting `atop.executable-path`.